assets/styl/_modules/_form.styl

```diff
@@ -0,0 +1,59 @@
+// Forms widgets
+//
+// Colette provides styles for:
+// - Fields descriptions (with valid / invalid color indication)
+// - Valid / Invalid fields
+// - Also a special style for a flex parent with 3 children (label, infolink, input 100%)
+//
+// Note: you can also use Alerts components to handle success or errors messages.
+//
+// Weight: 2
+//
+// Markup: form.twig
+//
+// Styleguide: Components.Forms.Form
+.form
+    &-field-widget
+        // display a widget for form element that must be filled with text content
+        // e.g. input[text|password|email|url]
+
+        label
+            display block
+
+        ul
+            margin 0
+
+        //special style for a flex parent with 3 children (label, infolink, input 100%)
+        &-flex3
```

Comment:
Olivia asked me to create a modifier because using it by default would impact something else maybe. And I think it no good to use flex by default because most of the time it will not be used (in "form-field-widget")
Then, I indicated 3 elements because I don't know how it will behave with more. I don't know flex that much. 
How can we be sure of it's behavior if there is more elements inside ?